scene/gui/line_edit.cpp

```diff
@@ -624,7 +624,12 @@ void LineEdit::gui_input(const Ref<InputEvent> &p_event) {
 			int prev_len = text.length();
 			insert_text_at_caret(ucodestr);
 			if (text.length() != prev_len) {
-				_text_changed();
+				if (!text_changed_dirty) {
+					if (is_inside_tree()) {
+						MessageQueue::get_singleton()->push_call(this, "_text_changed");
```

Comment:
```suggestion
						callable_mp(this, &LineEdit::_text_changed).call_deferred();
```
Would be best to use Callable here, see https://github.com/godotengine/godot/pull/86301, better to not risk conflicts between these PRs and just use this here